### PR TITLE
Remove CpsAssetPageMain, move logic to /pages

### PR DESCRIPTION
**Pull request by chris-hinds**
No issue

**Overall change:** 
Restructure our code to allow us to more easily implement the differing behaviour for each CPS asset page; currently this is all managed in one place.

**Code changes:**
- _`CpsMap` created._
- _`CpsPgl` created._
- _`CpsSty` created._
- _`dirAlias.js` updated_.

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added labels to this PR for the relevant pod(s) affected by these changes
- [ ] I have assigned this PR to the Simorgh project

**Testing:**
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

**Comment by andrewscfc** [comment]
Progress so far with @FK78 

Manually tested the following pages and all look correct:
- MAP http://localhost:7080/afrique/monde-49500638
- STY http://localhost:7080/pidgin/world-23252817
- PGL http://localhost:7080/pidgin/sport-23252855

When attempting to commit, getting issues with linting around proptypes, didn't get around to solving these issues yet:
image

Can replicate this with `npm run test:lint`

**Comment by karinathomasbbc** [comment]
I was just wondering... do we need to have CPS in the filename? Do we even need to use the same abbreviations that CPS do - like MAP, STY and PGL? Could we make the following amendments:
CPSSTY -> Story
CPSMAP -> MediaAssetPage
CPSPGL -> PhotoGallery

**Comment by jamesdonoh** [comment]
> I was just wondering... do we need to have CPS in the filename? Do we even need to use the same abbreviations that CPS do - like MAP, STY and PGL?

@karinathomas I agree - this was already briefly discussed at https://github.com/bbc/simorgh/pull/5047#discussion_r364213486. Might be better to do the renaming in a separate PR though to make this one easier to finish/review.

File: src/app/pages/CpsMap/index.jsx
Comment (jamesdonoh) [inline src/app/pages/CpsMap/index.jsx]: Trivial point, but we could delete this blank line for consistency with Pgl and Sty? :-)

File: src/app/pages/CpsMap/index.jsx
Comment (jamesdonoh) [inline src/app/pages/CpsMap/index.jsx:105]: This timestamp (and its associated imports) seem like an obvious immediate candidate for DRYing up, if only to make these files shorter, especially as we now know the timestamp behaviour/appearance should be consistent across CPS page types. But could be done in a follow-up PR.

File: src/app/pages/CpsMap/index.test.jsx
Comment (jamesdonoh) [inline src/app/pages/CpsMap/index.test.jsx:10]: Is there any reason this is imported as `CpsMapPage` but the other two are imported as `CpsAssetPage`? Not a real problem but we could keep them consistent.

File: src/app/pages/CpsMap/index.test.jsx
Comment (jamesdonoh) [inline src/app/pages/CpsMap/index.test.jsx:117]: Please can you provide a little more context around these new sections, and the added tests below? They seem to duplicate some of the Cypress test code (e.g. https://github.com/bbc/simorgh/blob/latest/cypress/integration/pages/mediaAssetPage/tests.js#L4), is that intentional?

File: src/app/pages/CpsMap/index.test.jsx
Comment (jamesdonoh) [inline src/app/pages/CpsMap/index.test.jsx:1]: Just wanted to check, why is this now necessary in all three test files? Is there any reasonably practical alternative?

**Review by jamesdonoh — COMMENTED** [review]
Thank you for sorting out the problems with the tests Rowland. This is looking in good shape, I had a couple of questions below. Also definitely needs a review from the media team because of all the changes to the MAP tests.

File: src/app/pages/CpsMap/index.test.jsx
Comment (rhenshaw56) [reply src/app/pages/CpsMap/index.test.jsx:117]: @jamesdonoh I think this helps transforms text like [this](https://github.com/bbc/simorgh/blob/latest/data/pidgin/cpsAssets/23248703.json#L38) to optimo format (like  what the preprocessor does) just to assert that the transformed text is present on the page. 

I don't have aa detailed context about most of this tests as they were already present in the removed [CpsAssetPageMain tests](https://github.com/bbc/simorgh/blob/latest/src/app/containers/CpsAssetPageMain/index.test.jsx#L57-L70) as part of the MAP implementation, I just moved them over and ensured they were passing like before. cc @ryanmccombe

File: src/app/pages/CpsMap/index.test.jsx
Comment (rhenshaw56) [reply src/app/pages/CpsMap/index.test.jsx:1]: this was in the originally removed test file but It would be trivial to get rid of this and add  proper prop validations.

File: src/app/pages/CpsMap/index.jsx
Comment (jamesdonoh) [reply src/app/pages/CpsMap/index.jsx:105]: To be addressed in https://github.com/bbc/simorgh/issues/5227

File: src/app/pages/CpsMap/index.test.jsx
Comment (jamesdonoh) [reply src/app/pages/CpsMap/index.test.jsx:117]: As discussed on the Zoom call, I misunderstood here, I thought they were new tests/logic rather than tests that tests that have been relocated. Happy to leave this as-is unless anyone in the media team has any different views.

File: src/app/pages/CpsMap/index.test.jsx
Comment (jamesdonoh) [reply src/app/pages/CpsMap/index.test.jsx:1]: Hadn't spotted it was in the original test file, apologies.

**Comment by jamesdonoh** [comment]
Renaming will be addressed separately in https://github.com/bbc/simorgh/issues/5228

**Review by jamesdonoh — APPROVED** [review]
Provided the media team are happy, let's merge this to unblock metadata and pick up the improvements in https://github.com/bbc/simorgh/issues/5227 and https://github.com/bbc/simorgh/issues/5228

**Review by FK78 — APPROVED** [review]

**Review by andrewscfc — APPROVED** [review]
Looks good, thanks @rhenshaw56 for picking up such a tricky refactor 🙂

**Comment by andrewscfc** [comment]
Fyi, I'm looking to resolve conflicts following merge of https://github.com/bbc/simorgh/pull/5141

**Comment by andrewscfc** [comment]
Conflicts should now all be resolved, I manually checked:
- http://localhost:7080/pidgin/23248703 for MAP
- http://localhost:7080/pidgin/sport-23252855 for PGL 
- http://localhost:7080/pidgin/world-23252817 for STY

File: src/app/pages/CpsMap/index.jsx
Comment (karinathomasbbc) [inline src/app/pages/CpsMap/index.jsx:51]: The following code is missing: https://github.com/bbc/simorgh/blob/cf7916fc34f067dd4289eb41ca994f86138d3041/src/app/containers/CpsAssetPageMain/index.jsx#L39

File: src/app/pages/CpsMap/index.jsx
Comment (karinathomasbbc) [inline src/app/pages/CpsMap/index.jsx]: Code missing: https://github.com/bbc/simorgh/blob/cf7916fc34f067dd4289eb41ca994f86138d3041/src/app/containers/CpsAssetPageMain/index.jsx#L24

**Review by karinathomasbbc — CHANGES_REQUESTED** [review]

File: src/app/pages/CpsMap/index.jsx
Comment (karinathomasbbc) [inline src/app/pages/CpsMap/index.jsx:81]: Missing: https://github.com/bbc/simorgh/blob/cf7916fc34f067dd4289eb41ca994f86138d3041/src/app/containers/CpsAssetPageMain/index.jsx#L69

File: src/app/pages/CpsMap/index.jsx
Comment (karinathomasbbc) [inline src/app/pages/CpsMap/index.jsx:89]: Missing: https://github.com/bbc/simorgh/blob/cf7916fc34f067dd4289eb41ca994f86138d3041/src/app/containers/CpsAssetPageMain/index.jsx#L77

**Review by karinathomasbbc — CHANGES_REQUESTED** [review]

**Review by karinathomasbbc — APPROVED** [review]

**Comment by karinathomasbbc** [comment]
There does seem to be a lot of duplicate code within PGL/STY/MAP - definitely keen that this is addressed in a future PR.

**Comment by andrewscfc** [comment]
> There does seem to be a lot of duplicate code within PGL/STY/MAP - definitely keen that this is addressed in a future PR.

Hey @karinathomasbbc , I think the idea is this gives us the opportunity to diverge between the different page types as we implement, PGLs and STYs for production. We definitely want to identify shared logic where we can at a later point, any ideas about how we make sure this doesn't get forgotten?

**Pull request merged**
